### ISSUE TITLE
CSS issue in CP when a Section is the first field on a page

### ISSUE THREAD

**Issue by bgarrant** [statamic/v2-hub#1865]
I have a fieldset that uses a hero_image partial.  

```
taxonomies: false
title: Default Page
fields:
  hero_image_partial:
    type: partial
    fieldset: hero_image
  long_form_content_partial:
    type: partial
    fieldset: long_form
```

In the partial there is a section field 

```
taxonomies: false
title: Hero Image
hide: true
fields:
  hero_image_section:
    type: section
    display: Hero Image
  hero_image:
    type: assets
    display: Hero Image
```

and the section field is overlapping the Title field when viewed on a desktop.  Looks like a CSS issue.

![screen shot 2018-04-02 at 10 21 20 pm](https://user-images.githubusercontent.com/2436728/38225804-a5e4f2e8-36c4-11e8-9899-e0387e2f86d4.png)

**Comment by jasonvarga** [comment]
Fixed for 2.9

**Comment by lukeagsmith** [comment]
Just in case that isn't caught as part of the fix, on narrow viewports on the home page the title field is concealed entirely.

![screen shot 2018-04-25 at 13 31 53](https://user-images.githubusercontent.com/16148258/39245962-1658fdb0-488d-11e8-9917-46d7227f9497.png)
